src/bikeracks/__init__.py

```diff
@@ -1,7 +1,12 @@
 # __init__ file contains the application factory and tells
 # Python that bikeracks should be treated as a package
 import os
-from flask import Flask, render_template
+from flask import Flask, render_template, request
+import csv
+from time import time
+from . import db
+from . import bikes
+from . import votes
```

Comment:
This is fine, but the usual convention is python standard library modules first, then libraries being used, then your own modules, like so:

```python
import os
import csv
import time

from flask import Flask, render_template, request

from . import db
from . import bikes
from . import votes
```

Reply:
Done